x-pack/plugins/maps/public/connected_components/toolbar_overlay/set_view_control/set_view_control.tsx

```diff
@@ -42,8 +74,91 @@ export class SetViewControl extends Component<Props, State> {
     lat: 0,
     lon: 0,
     zoom: 0,
+    coord: DEFAULT_SET_VIEW_COORDINATE_SYSTEM,
+    mgrs: "",
+    utm: {}
   };
 
+  static convertLatLonToUTM(lat, lon) {
```

Comment:
Can you make this and the other static methods into standalone functions, rather than adding static methods to the component?